### ISSUE TITLE
High CPU utilization

### ISSUE THREAD

**Issue by mikesir87** [sanex3339/laravel-5-angular-2-example#11]
When running `npm run dev`, I see that the `node` process is pretty much sitting at 100% at all times.  This is true even when Webpack has finished and it's just watching for changes.  Any pointers?  (Still new to Webpack, so not 100% sure where to start)

**Comment by sanex3339** [comment]
`npm run dev` runs `webpack --watch`. I dont't see any problems with webpack condig

**Comment by mikesir87** [comment]
Yeah. I didn't see anything odd with the config either. Wasn't sure if you had seen anything yet or not.

I ran `fs_usage` and `dtruss` (on a Mac) and saw that the `node` process is just hammering away trying to poll the project.  Since it has a ton of `node_modules`, all of the Composer dependencies, etc., it's just going nuts.  Looking around, it says I should be able to install `fsevents`, which Webpack will pick up and use.  But... I haven't been successful with that yet.  I'll do some more digging again soon.
